recipe-server/client/control_new/components/common/ShieldIdenticon.js

```diff
@@ -3,52 +3,51 @@ import PropTypes from 'prop-types';
 
 import Genome from 'control_new/utils/Genome';
 
-// Solarized colors
-export const colors = [
-  '#002b36', // base03
-  '#073642', // base02
-  '#586e75', // base01
-  '#657b83', // base00
-  '#839496', // base0
-  '#93a1a1', // base1
-  '#eee8d5', // base2
-  '#fdf6e3', // base3
-  '#b58900', // yellow
-  '#cb4b16', // orange
-  '#dc322f', // red
-  '#d33682', // magenta
-  '#6c71c4', // violet
-  '#268bd2', // blue
-  '#2aa198', // cyan
-  '#859900', // green
+
+const emojis = [
+  '😄', '😃', '😀', '😊', '😉', '😍', '😘', '😚', '😗', '😙', '😜', '😝', '😛', '😳', '😁', '😔',
+  '😌', '😒', '😞', '😣', '😢', '😂', '😭', '😪', '😥', '😰', '😅', '😓', '😨', '😱', '😠', '😡',
+  '😤', '😖', '😆', '😋', '😷', '😎', '😴', '😵', '😲', '😟', '😦', '😧', '😈', '👿', '😮', '😬',
+  '😐', '😯', '😶', '😇', '😏', '😑', '👼', '😺', '😻', '😽', '😼', '🙀', '😿', '😹', '😾', '👹',
+  '👺', '🙈', '🙉', '🙊', '💀', '👽', '💩', '🔥', '✨', '🌟', '💫', '💥', '💦', '💧', '💤', '👂',
+  '👀', '👃', '👅', '👄', '👍', '👎', '👌', '👊', '✊', '👋', '✋', '👐', '👆', '🙌', '🙏', '👏',
+  '💪', '💃', '🎩', '👑', '👒', '👟', '👞', '👡', '👠', '👢', '💼', '👜', '👝', '👛', '👓', '🎀',
+  '🌂', '💄', '💛', '💙', '💜', '💚', '💔', '💗', '💓', '💕', '💖', '💞', '💘', '💌', '💋', '💍',
+  '💎', '👣', '🐶', '🐺', '🐱', '🐭', '🐹', '🐰', '🐸', '🐯', '🐨', '🐻', '🐷', '🐽', '🐮', '🐗',
+  '🐵', '🐒', '🐴', '🐑', '🐘', '🐼', '🐧', '🐦', '🐤', '🐥', '🐣', '🐔', '🐍', '🐢', '🐛', '🐝',
+  '🐜', '🐞', '🐌', '🐙', '🐚', '🐠', '🐟', '🐬', '🐳', '🐋', '🐄', '🐏', '🐀', '🐃', '🐅', '🐇',
+  '🐉', '🐎', '🐐', '🐓', '🐕', '🐖', '🐁', '🐂', '🐲', '🐡', '🐊', '🐫', '🐪', '🐆', '🐈', '🐩',
+  '🐾', '💐', '🌸', '🌷', '🍀', '🌹', '🌻', '🌺', '🍁', '🍃', '🍂', '🌿', '🌾', '🍄', '🌵', '🌴',
+  '🌲', '🌳', '🌰', '🌱', '🌼', '🌐', '🌞', '🌝', '🌚', '🌜', '🌛', '🌙', '🌍', '🌎', '🌏', '⭐',
+  '⛅', '⛄', '🌀', '💝', '🎒', '🎓', '🎏', '🎃', '👻', '🎄', '🎁', '🎋', '🎉', '🎈', '🔮', '🎥',
+  '📷', '📹', '📼', '💿', '📀', '💽', '💾', '💻', '📱', '📞', '📟', '📠', '📡', '📺', '📻', '🔊',
+  '🔔', '📢', '⏳', '⏰', '🔓', '🔒', '🔏', '🔐', '🔑', '🔎', '💡', '🔦', '🔆', '🔅', '🔌', '🔋',
+  '🔍', '🛁', '🛀', '🚿', '🚽', '🔧', '🔨', '🚪', '💣', '🔫', '🔪', '💊', '💉', '💰', '💸',
+  '📨', '📬', '📌', '📎', '📕', '📓', '📚', '📖', '🔬', '🔭', '🎨', '🎬', '🎤', '🎵', '🎹', '🎻',
+  '🎺', '🎷', '🎸', '👾', '🎮', '🃏', '🎲', '🎯', '🏈', '🏀', '⚽', '🎾', '🎱', '🏉', '🎳', '⛳',
+  '🚴', '🏁', '🏇', '🏆', '🎿', '🏂', '🏄', '🎣', '🍵', '🍶', '🍼', '🍺', '🍻', '🍸', '🍹', '🍷',
+  '🍴', '🍕', '🍔', '🍟', '🍗', '🍤', '🍞', '🍩', '🍮', '🍦', '🍨', '🍧', '🎂', '🍰', '🍪', '🍫',
+  '🍬', '🍭', '🍯', '🍎', '🍏', '🍊', '🍋', '🍒', '🍇', '🍉', '🍓', '🍑', '🍌', '🍐', '🍍', '🍆',
+  '🍅', '🌽', '🏠', '🏡', '⛵', '🚤', '🚣', '🚀', '🚁', '🚂', '🚎', '🚌', '🚍', '🚙', '🚘', '🚗',
+  '🚕', '🚖', '🚛', '🚚', '🚨', '🚓', '🚔', '🚒', '🚑', '🚐', '🚲', '🚜', '💈', '🚦', '🚧', '🏮',
+  '🎰', '🗿', '🎪', '🎭', '📍', '🚩', '💯',
```

Comment:
^ Just kidding, since we're using template strings I also have to filter out `\n`s and empty strings:

```
[...`😗😙
😜😝😛`].filter(emo => emo.indexOf(/\n/) > -1 || emo.trim());
```